distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -480,6 +486,256 @@ public void testBasicRestAccess() throws Exception {
         getSecurityPolicy().waitForGuestAuthReady(url);
     }
 
+    @Test(expected = NoHttpResponseException.class)
+    public void testBasicAuthOverHttp() throws Exception {
+        String url = SERVICE_ROOT.getUrl()
+                .replace("https", "http") + "/catalog/query?q=*&src=local";
+
+        configureRestForBasic(SDK_SOAP_CONTEXT);
+
+        waitForSecurityHandlers(url);
+
+        //try basic auth over http
+        given().auth()
```

Comment:
I think the same applies to all of these tests. Alternatively, put them in their own class and clean up in the teardown method. Now that we have reduced the number of itest containers, we should avoid continuing to bloat the existing classes.